plutus-benchmark/nofib/test/Spec.hs

```diff
@@ -43,6 +47,7 @@ testClausify = testGroup "clausify"
                , testCase "formula4" $ mkClausifyTest Clausify.F4
                , testCase "formula5" $ mkClausifyTest Clausify.F5
                , Tx.fitsInto "formula1 (size)" (Clausify.mkClausifyCode Clausify.F1) 5190
+               , runTestNested $ Tx.goldenBudget "formulaBudget" $ Clausify.mkClausifyCode Clausify.F1
```

Comment:
When i run these it shows me the AST size but not the budget, presumably because `Tx.goldenBudget` and `Tx.fitsInto` do different things. Would it be feasible/sensible/worthwhile to get it to show us the budget too?

![nofib-tests-2](https://user-images.githubusercontent.com/2124565/152260181-2d90f76d-7723-433a-8471-2dd8ef8ef703.png)

Reply:
Also maybe call it `formula1 (budget)` etc. for consistency.
... or would that require the golden file to have a ridiculous name?

Reply:
> Would it be feasible/sensible/worthwhile to get it to show us the budget too?

Possibly. I think the other one does it because it's a custom tasty test type, but the golden tests already have one like that from a library, so I don't know if I can override it...

> or would that require the golden file to have a ridiculous name?

exactly :(

Reply:
Boooo!

Reply:
Maybe I can fix that somehow...

Reply:
Hmm, not easy to mess with the golden test stuff, I can do it if I put stuff in subdirectories, which might be okay.